.github/workflows/pr-check.yml

```diff
@@ -121,7 +121,7 @@ jobs:
         shell: bash
 
       - name: Run TS tests
-        uses: GabrielBB/xvfb-action@v1.6
+        uses: GabrielBB/xvfb-action@v1.7
```

Comment:
This is the latest version of `GabrielBB/xvfb-action` at the time of this writing.